### PR TITLE
pa11y test added

### PR DESCRIPTION
**Pull request by mariabg**
(No Pa11y test errors found, yaaay! 🎉 )

### CODE REVIEW

File: main.scss [hunk 1]
```diff
@@ -35,12 +35,25 @@
 			color: oColorsGetColorFor('o-footer-dark-link', 'text');
 
 			&:hover,
-			&:focus {
+			&:focus, {
```
Comment (onishiweb) [inline main.scss]: extra comma, will break the focus styles

File: main.scss [hunk 2]
```diff
@@ -35,12 +35,25 @@
...
 				color: oColorsGetColorFor('o-footer-dark-link-hover', 'text');
 			}
 		}
 	}
 
+	.o-footer--theme-dark-hover {
```
Comment (onishiweb) [inline main.scss]: Are these styles just for the pa11y test? If so, should go into the demo CSS separately and not be named using the `o-footer` classes style.

File: main.scss [hunk 3]
```diff
@@ -54,6 +67,15 @@
 		}
 	}
 
+	.o-footer--theme-light-hover {
```
Comment (onishiweb) [inline main.scss]: As above

File: main.scss [hunk 4]
```diff
@@ -165,12 +187,14 @@
 		padding-top: $o-footer-spacing-unit - 5;
 		padding-bottom: $o-footer-spacing-unit - 5;
 
-		.o-footer--theme-dark & {
+		.o-footer--theme-dark &,
+		.o-footer--theme-dark-hover {
```
Comment (onishiweb) [inline main.scss]: hover styles again

File: origami.json [hunk 5]
```diff
@@ -40,6 +40,12 @@
 			"template": "demos/src/simple-footer.mustache",
 			"expanded": true,
 			"description": ""
+		},
+		{
+			"name": "pa11y",
+			"template": "demos/src/pa11y.mustache",
+			"expanded": true,
```
Comment (onishiweb) [inline origami.json]: Should be hidden not expanded. And all the other expanded properties should be removed.

File: demos/src/pa11y.mustache [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+		</div>
+	</div>
+	<div class="o-footer__brand">
+		<div class="o-footer__container">
+			<div class="o-footer__brand-logo"></div>
+		</div>
+	</div>
+</footer>
+
+<footer class="o-footer o-footer--theme-light-hover" data-o-component="o-footer" data-o-footer--no-js>
```
Comment (onishiweb) [inline demos/src/pa11y.mustache]: Commented in the CSS below but if this is just a style for the pa11y test it shouldn't use the `o-footer` style classnames

**Comment by onishiweb** [comment]
Few comments but looks good :)

File: origami.json (hunk 5 above)
Comment (mariabg) [inline origami.json]: sure! I wrote this test few weeks ago and I didn't realise that it had the old format, thanks!

**Pull request merged**